templates/index.html

```diff
@@ -0,0 +1,74 @@
+<html>
+<head>
+    <title> Index of {{ path }} </title>
+    <link rel="stylesheet" href="{{url_for('static', filename='/common/css/fileindex.css')}}" type="text/css">
+</head>
+<body>
+    <h1> Index of {{ path }} </h1>
+    <table id="indexlist">
+        <tr class="indexhead">
+            <th class="indexcolicon">
+                <img src="{{url_for('static', filename='/icons/blank.png')}}" alt="[ICO]"/>
+            </th>
+            <th class="indexcolname">
+                <a> Name </a>
+            </th>
+            <th class="indexcollastmod">
+                <a> Last modified </a>
+            </th>
+            <th class="indexcolsize">
+                <a> Size </a>
+            </th>
+        </tr>
+        <tr class="indexbreakrow">
+            <th colspan="5">
+                <hr>
+            </th>
+        </tr>
+        <tr class="even">
+            <td class="indexcolicon">
+                <img src="{{ url_for('static', filename='/icons/back.png') }}" alt="[PARENTDIR]">
+            </td>
+            <td class="indexcolname">
+                <a href="{{ url_for('s3_view', subpath=parent_path) }}"> Parent Directory </a>
+            </td>
+            <td class="indexcollastmod">&nbsp;</td>
+            <td class="indexcolsize"></td>
+        </tr>
+        {% for folder in folders %}
+            {% if loop.index % 2 == 0 %}
+                <tr class="even">
+            {% else %}
+                <tr class="odd">
+            {% endif %}
+                <td class="indexcolicon">
+                    <img src="{{ url_for('static', filename='/icons/folder.png') }}" alt="[DIR]">
+                </td>
+                <td class="indexcolname">
+                        {% set folder_path = path + folder %}
+                        <a href="{{ url_for('s3_view', subpath=folder_path, type='folder') }}"> {{ folder }} </a>
+                </td>
+                <td class="indexcollastmod">{{ folders[folder].strftime("%Y-%m-%d %H:%M:%S") }}</td>
+                <td class="indexcolsize"></td>
+            </tr>
+        {% endfor %}
+        {% for file in files %}
+            {% if loop.index % 2 == 0 %}
+                <tr class="even">
+            {% else %}
```

Comment:
Look at https://jinja.palletsprojects.com/en/3.0.x/tricks/#alternating-rows

Reply:
We can use pure CSS for that, like [here](https://tkn.me/tarantool-github-statistics-20200818/):

```css
table tr:nth-child(odd) td {
  background-color: #333;
}

table tr:nth-child(even) td {
  background-color: #555;
}
```

Reply:
Done. I used it. See the last patch.